sky/backends/backend_utils.py

```diff
@@ -2261,8 +2264,11 @@ def is_controller_up(
         # unnecessary costly refresh when the controller is already stopped.
         # This optimization is based on the assumption that the user will not
         # start the controller manually from the cloud console.
+        # The acquire_lock_timeout is set to 1 second to avoid hanging the
```

Comment:
A bit confused why this says "set to 1 second" and 0 later.

Reply:
Good catch! Changed to 0.